flow/connectors/bigquery/bigquery.go

```diff
@@ -1088,6 +1088,16 @@ func (c *BigQueryConnector) SetupNormalizedTables(
 ) (*protos.SetupNormalizedTableBatchOutput, error) {
 	tableExistsMapping := make(map[string]bool)
 	for tableIdentifier, tableSchema := range req.TableNameSchemaMapping {
+		table := c.client.Dataset(c.datasetID).Table(tableIdentifier)
```

Comment:
Is this code movement only for optimization purposes & not correctness purposes?

Reply:
only for optimization